maintainer-policies-and-guidelines.md

```diff
@@ -0,0 +1,29 @@
+# Maintainer Policies and Guidelines
+
+This document aims to clarify hard policies that Maintainers have to follow as well as less-enforced guidelines that are recommended to be followed. The goal is to help Maintainers understand what is expected of them and what actions should be done to reduce the chances of conflict within the Maintainer team.
+
+## Policies
+
+- `c` tags in 1.21+ NeoForge should be kept in sync with Fabric's `c` tags as close as possible. New `c` tags or changes to existing `c` tag should by ran by Fabric to ensure they will agree to make the same changes. `c` tags are to be a standard between loaders with Fabric being our main partner in this effort.
+
+  - Failure to meet this expectation will result in incident being reviewed to understand how we failed to communicate with Fabric. The PR in question could be reverted out of NeoForge if Fabric rejects the tag changes. Will be handled on a case-by-case basis.
+
+## Guidelines
+
+- Breaking change window for the NeoForge project will generally aim to last only 1 month after a significant Minecraft version.
+
+  - Hotfix Minecraft versions or very small Minecraft versions will not reset the breaking change window.
+
+  - This window is flexible and may be longer if there is significant breaking change PRs that need to be released for that Minecraft version but not yet ready. The 1 month timeframe is just a goal we would like to achieve but we understand it is not always possible or optimal.
+
+- When making a "big-deal" PR, please post a link to the PR in the NeoForge Discord's maintenance-talk channel, ping the maintainer role, and leave a short sentence on what the PR is about.
```

Comment:
This entire paragraph and the sublist can be replaced by something much shorter. For example `Maintainers should ping @Maintainers on discord before merging important PRs. What is considered important enough to warrant a ping is left to maintainer discretion, but it is better to ping too often rather than not enough.`